src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackTransportTest.php

```diff
@@ -239,4 +239,32 @@ public function testSendIncludesContentTypeWithCharset()
 
         $transport->send(new ChatMessage('testMessage'));
     }
+
+    public function testSendWithErrorsIncluded()
+    {
+        $response = $this->createMock(ResponseInterface::class);
+
+        $response->expects($this->exactly(2))
+            ->method('getStatusCode')
+            ->willReturn(200);
+
+        $response->expects($this->once())
+            ->method('getContent')
+            ->willReturn(json_encode([
+                'ok' => false,
+                'error' => 'invalid_blocks',
+                'errors' => ['no more than 50 items allowed [json-pointer:/blocks]'],
```

Comment:
Are they talking about our option blocks? Should we also check the count of 50 beforehand to provide a better DX?

Reply:
Its just one of many possible errors.. I could make a follow up PR for block limiting

Reply:
Thanks I would appreciate sich PRs if we can give more infos beforehand to the developer 👍🏻